moveit_ros/planning_interface/CMakeLists.txt

```diff
@@ -34,46 +36,35 @@ elseif()
   set(BOOST_PYTHON_COMPONENT python${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR})
 endif()
 
-#catkin_lint: ignore_once duplicate_find
-find_package(Boost REQUIRED COMPONENTS
```

Comment:
You just removed these packages, shouldn't we do as above and require them?

Reply:
True, I just looked at the doc and we should do it as above... my bad